spec/ruby/optional/capi/ext/exception_spec.c

```diff
@@ -36,6 +36,19 @@ VALUE exception_spec_rb_set_errinfo(VALUE self, VALUE exc) {
   return Qnil;
 }
 
+VALUE exception_spec_rb_syserr_new(VALUE self, VALUE num, VALUE msg) {
+  char *cstr;
```

Comment:
This should be initialized to NULL explicitly (in C otherwise it's a random value).
@andrykonchin Could you do that?